components/raftstore/src/store/config.rs

```diff
@@ -60,9 +60,11 @@ pub struct Config {
     // When the entry exceed the max size, reject to propose it.
     pub raft_entry_max_size: ReadableSize,
 
-    // Interval to gc unnecessary raft log (ms).
+    // Interval to gc unnecessary raft log (s).
```

Comment:
```suggestion
    // Interval to gc unnecessary raft log.
```
`ReadableDuration` doesn't have an unit.